contract/proxy-oracle/soroban/scripts/dry_run_deploy.py

```
@@ -0,0 +1,190 @@
+            )
+        else:
+            print(f"  [OK] {key}: sha256 verified ({actual_sha[:16]}...)")
+
+    return errors
+
+
+# ---------------------------------------------------------------------------
+# Report
+# ---------------------------------------------------------------------------
+
+def format_report(manifest: dict, root: Path, errors: list[str]) -> str:
```

Ruff (0.15.15)

[warning] Unused function argument: `root`

(ARG001)